polyfills/String/prototype/padEnd/polyfill.js

```diff
@@ -0,0 +1,17 @@
+(function () {
+	Object.defineProperty(String.prototype, 'padEnd', {
+		configurable: true,
+		enumerable: false,
```

Comment:
It stops the method from appearing in the results of any enumeration such as `for..in`.

```js
➜  ~ node
> for (var a in String.prototype) {
...   console.log(a);
... }
undefined
>
> String.prototype.padEnd = function padEnd (targetLength, padString) {
...   targetLength = targetLength | 0;
...   if (targetLength <= this.length) return String(this);
...   padString = String(padString || " ");
...   var repeat = Math.ceil((targetLength - this.length) / padString.length);
...   while (repeat--) {
.....     padString += padString;
.....   }
...   return String(this) + padString.substr(0, targetLength - this.length);
... };
[Function: padEnd]
>
> for (var b in String.prototype) {
...   console.log(b);
... }
padEnd
undefined
>
> delete String.prototype.padEnd;
true
>
> Object.defineProperty(String.prototype, 'padEnd', {
...   configurable: true,
...   enumerable: false,
...   value: function padEnd (targetLength, padString) {
.....     targetLength = targetLength | 0;
.....     if (targetLength <= this.length) return String(this);
.....     padString = String(padString || " ");
.....     var repeat = Math.ceil((targetLength - this.length) / padString.length);
.....     while (repeat--) {
.......       padString += padString;
.......     }
.....     return String(this) + padString.substr(0, targetLength - this.length);
.....   },
...   writable: true
... });
[String: '']
>
> for (var c in String.prototype) {
...   console.log(c);
... }
undefined
>
```